cucumber/api/features/resource_list_by_role.feature

```diff
@@ -27,3 +27,7 @@ Feature: List resources for another role
     When I successfully GET "/resources?acting_as=cucumber:user:alice"
     Then the resource list should contain "variable" "dev/dev-var"
     And the resource list should not contain "variable" "prod/prod-var"
+
+  Scenario: Attempting to retrieve the resource list for a different role but without giving the account in the ID results in a 403
+  When I GET "/resources?acting_as=user:alice"
+  Then the HTTP response status code is 403
```

Comment:
@izgeri is there also a log message that we want to verify here?

Reply:
do you have an example cuke that you're thinking of? I don't see something like that in the other cukes around resources, so I'm not sure exactly what you mean.

Reply:
ok i ran the test and i see that we don't log anything regarding this error (other than the 403). In any case, we have other cukes that verify log messages, for example [this one](https://github.com/cyberark/conjur/blob/master/cucumber/authenticators_oidc/features/authn_oidc.feature#L93)